scripts/tasks.sh

```diff
@@ -490,32 +490,32 @@ spec:
   template:
     spec:
       tolerations:
-        - key: node-role.kubernetes.io/master
+        - key: node-role.kubernetes.io/control-plane
           operator: Exists
       containers:
         - name: rook-ceph-operator
           env:
             - name: DISCOVER_TOLERATION_KEY
-              value: node-role.kubernetes.io/master
+              value: node-role.kubernetes.io/control-plane
             - name: CSI_PROVISIONER_TOLERATIONS
               value: |
-                - key: node-role.kubernetes.io/master
+                - key: node-role.kubernetes.io/control-plane
                   operator: Exists
             - name: CSI_PLUGIN_TOLERATIONS
               value: |
-                - key: node-role.kubernetes.io/master
+                - key: node-role.kubernetes.io/control-plane
                   operator: Exists
 EOF
     fi
 
     # EKCO tolerations
     if kubernetes_resource_exists kurl deployment ekc-operator; then
-        kubectl -n kurl patch deployment ekc-operator --type=merge -p '{"spec":{"template":{"spec":{"tolerations":[{"key":"node-role.kubernetes.io/master","operator":"Exists"}]}}}}'
+        kubectl -n kurl patch deployment ekc-operator --type=merge -p '{"spec":{"template":{"spec":{"tolerations":[{"key":"node-role.kubernetes.io/control-plane","operator":"Exists"}]}}}}'
```

Comment:
Rather than remove the existing `node-role.kubernetes.io/master` toleration can we append `node-role.kubernetes.io/control-plane` as an additional toleration?